core/i18n/resources/en.js

```diff
@@ -342,6 +342,14 @@ export const enTranslation = {
     basic: 'Basic',
     advanced: 'Advanced',
     validations: 'Validations',
+    variable: 'Variable',
+    variables: 'Variables',
```

Comment:
preferably you can use i18next convention for plural with _plural suffix. this would become:
```
variable: 'Variable',
variable_plural: 'Variables',
```